lib/googleanalytics.js

```diff
@@ -21,15 +24,70 @@ module.exports = {
     ga('create', process.env.GA_ACCOUNT, 'auto');
     ga('send', 'pageview');
   },
+
   /**
    * pageview:
    * a proxy to basic GA pageview tracking
-   * @param  {[type]} path [description]
-   * @return {[type]}      [description]
+   * @param  {String} path - the current page page e.g. '/about'
    */
   pageview: function (path) {
     if (typeof ga === 'function') {
       ga('send', 'pageview', path);
     }
+  },
+
+  /**
+   * modalview:
+   * a proxy to basic GA pageview tracking
+   * @param  {String} modalName e.g. 'add-or-edit-club'
+   */
+  modalview: function (modalName) {
+    path = '/modal/' + modalName;
```

Comment:
Nope, it'd just be the case that `/mozilla-web-clubs/add` would show you the "darkened" page in the background w/ the modal focused in front of it, and anything that currently opens the modal via JS would instead just be a clickable link to `/mozilla-web-clubs/add`. The fancypants router would then trigger the transition to the modal.

If that is really weird and nobody else does it then we don't have to... I just like every "thing" on a page to have a URL I guess. It's also a bit annoying to implement in the current architecture so it wouldn't be implemented in v1.